lua/ulx/modules/sh/timed_gag.lua

```diff
@@ -243,23 +243,34 @@ end
 
 -- ULX COMMAND SETUP --
 
```

Comment:
```suggestion
local function removeTimeGag(ply)
    removeGagFromDatabase( ply )
    ungagPlayer( ply )
end
```

Reply:
Are you suggesting this be its own function rather than an opposite function?